telemetry/definitions/commonDefinitions.json

```
@@ -3788,7 +3936,7 @@
         {
             "name": "codeTransform_projectDetails",
             "passive": true,
-            "description": "Transform initiates project validation before user interaction and we want to gather project details then.",
+            "description": "Transform initiates project validation before user interaction and we want to gather project details then. (Deprecated - use codeTransform_validateProject)",
```

Let's make sure the developer cannot miss it:

```suggestion
            "description": "Deprecated - use codeTransform_validateProject",
```